index.html

```diff
@@ -11,22 +11,200 @@
 <body>
     <header class="navbar">
```

Comment:
- [ ] In the desktop view the hamburger menu should not be displayed, and the nav bar should contain the navigation menu (nav links)

|Your design|Figma design|
|-|-|
|![Screenshot_20220609_162844](https://user-images.githubusercontent.com/25227022/172886055-0197ff72-718f-4e0f-8949-9de3894645ab.png)|![Screenshot_20220609_162930](https://user-images.githubusercontent.com/25227022/172886105-f7d0341f-9f59-4aeb-a7f7-35fea66c01a3.png)|